lib/ts/recipe/totp/index.ts

```diff
@@ -0,0 +1,86 @@
+/* Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import TotpRecipe from "./recipe";
+import SuperTokensError from "./error";
+
+export default class Wrapper {
+    static init = TotpRecipe.init;
+
+    static Error = SuperTokensError;
+
+    static createDevice(input: {
+        userId: string;
+        deviceName: string;
+        skew?: number;
+        period?: number;
+        userContext?: any;
+    }) {
+        return TotpRecipe.getInstanceOrThrowError().recipeInterfaceImpl.createDevice({
+            userContext: input.userContext === undefined ? {} : input.userContext,
+            ...input,
```

Comment:
userContext has to go after `...input` in all these functions

Reply:
Done